### PR TITLE
Add Support for New MVLayer Times Parameter

### PR DESCRIPTION
**Pull request by htran84**
Primary changes in this Pull Request:

- Allow setting the times parameter when setting result WMS layers
- Fix results controller to add geojson layer to CesiumMapViews correctly.

Please review the checklist before submitting the Pull Request:

- [x] Pull request has a meaning full name (not just the commit message from of your last commit)
- [x] Code has been linted using flake8
- [x] All methods have accurate Google-Style Docstrings
- [x] 100% test coverage for new content
- [x] Tests for each common use case (please don't write one test that covers all use cases)
- [ ] Bonus: Use TypeHints

Explain deviations from original design if applicable:

### CODE REVIEW

File: tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py [hunk 1]
```diff
@@ -140,6 +137,19 @@ def get_context(self, request, session, resource, context, model_db, workflow_id
...
             layer_groups.insert(0, results_layer_group)
 
+        if self.map_type == "cesium_map_view":
+            layers, entities = self.translate_layers_to_cesium(results_layers)
+            for layer in layers:
+                # Add layer to beginning the map's of layer list
+                map_view.layers.insert(0, layer)
+            for entity in entities:
+                # Add entity to beginning the map's of entity list
+                map_view.entities.insert(0, entity)
```
Comment (swainn) [inline tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py:147]: Let's use list addition to preserve the order of `layers`:

```python
map_view.layers = layers + map_view.layers
map_view.entities = entities + map_view.entities
```

The loop will end up reversing the order of the layers and entities.

File: tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py [hunk 2]
```diff
@@ -140,6 +137,19 @@ def get_context(self, request, session, resource, context, model_db, workflow_id
...
+        else:
+            for result_layer in results_layers:
+                # Add layer to beginning the map's of layer list
+                map_view.layers.insert(0, result_layer)
```
Comment (swainn) [inline tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py:151]: Same thing here:

```python
map_view.layers = result_layers + map_view.layers
```

**Review by swainn — CHANGES_REQUESTED** [review]

File: tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py (hunk 1 above)
Comment (htran84) [reply tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py:147]: That's how we had it before. I just don't want to change the behavior of the existing app.

File: tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py [hunk 3]
```diff
@@ -140,15 +140,11 @@ def get_context(self, request, session, resource, context, model_db, workflow_id
         if self.map_type == "cesium_map_view":
             layers, entities = self.translate_layers_to_cesium(results_layers)
             for layer in layers:
-                # Add layer to beginning the map's of layer list
-                map_view.layers.insert(0, layer)
+                map_view.layers = layers + map_view.layers
```
Comment (swainn) [inline tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py]: Remove for loop

**Review by swainn — APPROVED** [review]

**Pull request merged**